src/main/java/step4/InputRacingCar.java

```diff
@@ -0,0 +1,24 @@
+package step4;
+
+import java.util.Scanner;
+
+public class InputRacingCar {
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
정적상수로 초기화 해도 괜찮을 것 같습니다 😄 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```